src/public/js_native_ext_api.h

```diff
@@ -28,8 +28,71 @@ typedef struct napi_ext_ref__ *napi_ext_ref;
 // A callback to return buffer synchronously
 typedef void (*napi_ext_buffer_callback)(napi_env env, uint8_t const *buffer, size_t buffer_length, void *buffer_hint);
 
+// A callback to run task
+typedef void (*napi_ext_task_callback)(napi_env env, void *task_data);
+
+// A callback to schedule a task
+typedef void (*napi_ext_schedule_task_callback)(
+    napi_env env,
+    napi_ext_task_callback task_cb,
+    void *task_data,
+    uint32_t delay_in_msec,
```

Comment:
>uint32_t delay_in_msec, [](http://example.com/codeflow?start=4&length=23)

(nit) delay no longer needed, we could simplify this and NapiJSITaskRunner a little bit (but we could do it later as well).